core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -26,6 +26,7 @@ private CatalogProperties() {}
 
   public static final String CATALOG_IMPL = "catalog-impl";
   public static final String FILE_IO_IMPL = "io-impl";
+  public static final String DEFAULT_FILE_IO_IMPL = "org.apache.iceberg.io.ResolvingFileIO";
```

Comment:
I think there is a slight problem with moving the property here since I do not believe all catalogs use this as the DEFAULT_FILE_IO so it's a bit misleading.

Reply:
I see... I ripped this out of `RestSessionCatalog`, since I noticed there are no public constants there and I wanted the same semantics as `loadTable`. Is there a better place I can put this?

Reply:
You could just call it DEFAULT_REST_FILE_IO_IMPL